dashboard-config/pr-view-config.yml

```diff
@@ -361,4 +361,23 @@ sections:
       - value: '{{ userdata.count("Azure/publish-security-assessments","stale") }}'
         url: '{{ userdata.url("Azure/publish-security-assessments","stale") }}'
       - value: '{{ userdata.count("Azure/publish-security-assessments","closed") }}'
-        url: '{{ userdata.url("Azure/publish-security-assessments","closed") }}'
\ No newline at end of file
+        url: '{{ userdata.url("Azure/publish-security-assessments","closed") }}'
+    
+    - - value: 'arm-deploy' 
+      - value: '{{ userdata.count("Azure/arm-deploy","") }}'
+        url: '{{ userdata.url("Azure/arm-deploy","") }}'
+      - value: '{{ userdata.count("Azure/arm-deploy","idle") }}'
+        url: '{{ userdata.url("Azure/arm-deploy","idle") }}'
+      - value: '{{ userdata.count("Azure/arm-deploy","stale") }}'
+        url: '{{ userdata.url("Azure/arm-deploy","stale") }}'
+      - value: '{{ userdata.count("Azure/arm-deploy","closed") }}'
+        url: '{{ userdata.url("Azure/arm-deploy","closed") }}'
```

Comment:
Can you please have a blank line between values for both the actions?

Reply:
@kanika1894 Done